lib/LibManifestPlugin.js

```diff
@@ -28,6 +28,7 @@ LibManifestPlugin.prototype.apply = function(compiler) {
 			var manifest = {
 				name: name,
 				type: this.options.type,
+				extensions: compiler.options.resolve.extensions,
```

Comment:
Plugins are not allow to access this property. Just remove `extensions` from the manifest.